lambdas/service/app.py

```diff
@@ -179,37 +179,95 @@ def openapi():
 
 
 health_spec = {
+    'tags': ['Health'],
     'responses': {
         '200': {
-            'description': '200 response',
+            'description': 'Specified service component is reachable.',
             'content': {
                 'application/json': {
-                    'schema': {'$ref': '#/components/schemas/Empty'}
+                    'schema': {
+                        'type': 'object',
+                        'additionalProperties': {
+                            'type': 'object',
+                            'properties': {
+                                'up': {
+                                    'type': 'boolean'
+                                }
+                            },
+                            'additionalProperties': True
+                        },
+                        'properties': {
+                            'up': {
+                                'type': 'boolean'
+                            }
+                        }
+                    }
+                }
+            }
+        },
+        '503': {
+            'description': 'Specified service component is unreachable.',
+            'content': {
+                'application/json': {
+                    'schema': {
+                        'type': 'object',
+                        'additionalProperties': {
+                            'type': 'object',
+                            'properties': {
+                                'up': {
+                                    'type': 'boolean',
+                                    'default': False
+                                }
+                            },
+                            'additionalProperties': True
+                        },
+                        'properties': {
+                            'up': {
+                                'type': 'boolean',
+                                'default': False
```

Comment:
See if you can use the `enum` workaround described in https://github.com/OAI/OpenAPI-Specification/issues/1313